app/kuma-dp/cmd/run.go

```diff
@@ -243,6 +238,24 @@ func newRunCmd(opts kuma_cmd.RunCmdOpts, rootCtx *RootContext) *cobra.Command {
 				return err
 			}
 
+			go func() {
```

Comment:
why this goroutine started not immediately after `gracefulCtx, ctx := opts.SetupSignalHandler()` as it was before?

Reply:
I need an access to `dataplane` var

Reply:
It's just strange we do bootstrapRequest without any context. There is 10s timeout, but I think it'd be better to cancel the request if signal is received at that time

Reply:
Sort of but that's the API of `http.Client#Post` it does not take a context

Reply:
It seems like request itself should be with context like we do here https://github.com/kumahq/kuma/blob/master/app/kumactl/pkg/resources/request.go#L14

Reply:
done